crates/wash-cli/src/app/mod.rs

```diff
@@ -125,14 +125,18 @@ pub struct PutCommand {
 
 #[derive(Args, Debug, Clone)]
 pub struct GetCommand {
-    /// The name of the application to retrieve
+    /// The name of the application to retrieve. If left empty retrieves all the applications, same as `wash app list`
     #[clap(name = "name")]
-    app_name: String,
+    app_name: Option<String>,
 
     /// The version of the application to retrieve. If left empty, retrieves the latest version
     #[clap(name = "version")]
     version: Option<String>,
 
+    /// Enables Real-time updates, duration can be specified in ms or in humantime (eg: 5s, 2m, 15ms). Defaults to 1000 milliseconds.
+    #[clap(long,short, num_args = 0..=1, default_missing_value = "1000", value_parser = parse_watch_interval)]
+    pub watch: Option<std::time::Duration>,
+
```

Comment:
NIT: same note about the spacing in the comment

Also, wondering if we should default to milliseconds here or seconds since it's on human scale...